src/background/extensionController/states.js

```diff
@@ -65,6 +65,10 @@ export class StateFirefoxVPNEnabled extends FirefoxVPNState {
 
 /**
  * When Firefox VPN is Off
+ *
+ * bypassTunnel = true means the VPN client is still connected, but the extension is off.
+ * In this case, all traffic should be routed through the Firefox proxy (bypassTunnel = true),
+ * otherwise (bypassTunnel = false) the browser proxy is used if set, or direct if not.
```

Comment:
I think we might want to settle on some terminology here- I'm getting confused between instances of 'the proxy', 'Firefox proxy', 'Browser proxy', and 'local proxy'. I know Santiago is using 'Firefox proxy' in the ticket but how about we settle on `browser proxy` and `local proxy`?